packages/react-components/react-field/src/components/CheckboxField/CheckboxField.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import type { CheckboxProps } from '@fluentui/react-checkbox';
+import { Checkbox } from '@fluentui/react-checkbox';
+import { ForwardRefComponent } from '@fluentui/react-utilities';
+import type { FieldProps } from '../../Field';
+import { getFieldClassNames, renderField_unstable, useFieldStyles_unstable, useField_unstable } from '../../Field';
+
+export type CheckboxFieldProps = Omit<FieldProps<typeof Checkbox>, 'label'> & {
```

Comment:
Does `Switch` also need to implement this behavior since it also already has a `label` prop?

Reply:
I'm talking to Design about how the label prop of Switch should work. I logged this issue to implement the final behavior once we decide how it should work:
* #24685